forem/forem
# Implement Unified Embeds for Stackblitz URLs

## What type of PR is this? (check all applicable)


## Description
This PR implements the [unified embed tag](https://github.com/forem/forem/issues/15099) for Stackblitz embeds.

## Related Tickets & Documents
closes https://github.com/forem/forem/issues/15576

## QA Instructions, Screenshots, Recordings
Example Stackblitz URLs:
https://stackblitz.com/edit/web-platform-3tqbd4
https://stackblitz.com/edit/web-platform-3tqbd4?embed=1&file=index.html&hideExplorer=1&hideNavigation=1&theme=dark
https://stackblitz.com/edit/web-platform-3tqbd4?embed=1&file=index.html&theme=light

Example Stackblitz ID: web-platform-3tqbd4

- As a user, create a Stackblitz liquid tag using this format: `{% embed <stackblitz-url> %}`. The Liquid Tag should render properly.
- Also create Stackblitz liquid tag using the old method: `{% stackblitz <stackblitz-id> %}`. This should also work, as I am leaving the old implementation in place for now.

### UI accessibility concerns?
None

## Added/updated tests?


## [Forem core team only] How will this change be communicated?
Considering the project as a whole, I shall update the Liquid Tag documentation at the end. Since I am not changing the current implementation, the present documentation is still valid. Plus, I will avoid updating the docs piecemeal, and risk confusing users.

## [optional] Are there any post deployment tasks we need to perform?
None
